src/errors.ts

```diff
@@ -10,9 +10,24 @@ export class NoWorkspaceError extends Error {
     public message: string = localize('noWorkspaceError', 'You must have a workspace open to perform this operation.');
 }
 
+// use only when using the gitApi
```

Comment:
how about `handleGitErrors`

Just feels like functions should start with a verb instead of being a noun